client/components/follow-button/style.scss

```diff
@@ -1,3 +1,5 @@
+$follow-button-gray-disabled: lighten( $gray, 20% );
```

Comment:
There aren't any rules for it, but it's better to just stick to the original values. I don't think we ever really have custom variables outside the ones that are being shared globally. Lines `89` and `93` can just use `inherit` so we don't do much repitition.